app/webpack/stylesheets/_tables.scss

```diff
@@ -61,7 +61,7 @@ table {
       }
     }
 
-    &.cav_warning {
+    &.injection_warning {
```

Comment:
[ditto](https://github.com/ministryofjustice/Claim-for-Crown-Court-Defence/pull/3478/files#r477154369)